packages/components/pie-modal/test/visual/pie-modal.spec.ts

```diff
@@ -2,21 +2,21 @@ import { test } from '@sand4rt/experimental-ct-web';
 import percySnapshot from '@percy/playwright';
 import { PieModal } from '@/index';
 
-const propIsOpenValues = [true, false];
+const propIsOpenValues = [
+    { heading: 'Modal Header / isOpen = true', isOpen: true },
```

Comment:
whilst this is perfectly reasonable, if you need to label the tests I would suggest using the test component wrapper and providing a string label.

Entirely up to you but it might be nicer for consistency. Example can be found in the pie-button visual tests

Reply:
actually, I might've just been a bit expressive here and reverted that.  The tests are to make sure the component is open or not and I thought we should add the state to the heading, but the heading won't render anyway because the modal is closed and not in the DOM 😵‍💫 😆